### PR TITLE
Ensured all capability providers have friendly names

### PR DESCRIPTION
**Pull request by brooksmtownsend**
Signed-off-by: Brooks Townsend <brooks@cosmonic.com>

### CODE REVIEW

File: httpserver-rs/Makefile [hunk 1]
```diff
@@ -4,6 +4,7 @@
 PROJECT  = httpserver
 CAPABILITY_ID = wasmcloud:httpserver
 VENDOR   = "wasmCloud"
+NAME     = "HttpServer"
```
Comment (autodidaddict) [inline httpserver-rs/Makefile:7]: Suggest maybe "HTTP Server"

File: kvredis/Makefile [hunk 2]
```diff
@@ -3,6 +3,7 @@
 PROJECT = kvredis
 CAPABILITY_ID = wasmcloud:keyvalue
 VENDOR   = "wasmCloud"
+NAME     = "KeyValue Redis"
```
Comment (autodidaddict) [inline kvredis/Makefile:6]: How about Redis Key Value Store ?

File: sqldb-postgres/Makefile [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 PROJECT = sqldb-postgres
 CAPABILITY_ID = "wasmcloud:sqldb"
-NAME = "SqlDb-Postgres"
+NAME = "SqlDb Postgres"
```
Comment (autodidaddict) [inline sqldb-postgres/Makefile:5]: Maybe Postgres SQL Provider ?

**Review by autodidaddict — CHANGES_REQUESTED** [review]
Just some super minor requests to make the names a little more easily read in tooling output like wash

File: sqldb-postgres/Makefile (hunk 3 above)
Comment (brooksmtownsend) [reply sqldb-postgres/Makefile:5]: I like `Postgres SQL` better, but I've been keeping `provider` out of the names. Open to reconsider

Comment (autodidaddict) [reply sqldb-postgres/Makefile]: Agreed we probably shouldn't put the word provider in a providers name

**Review by autodidaddict — APPROVED** [review]

**Review by autodidaddict — APPROVED** [review]

**Pull request merged**